config/config.go

```diff
@@ -18,13 +18,21 @@ const (
 	ENV_IDENTITY_CERT_FILE        = "IDENTITY_CERT_FILE"
 	ENV_IDENTITY_PRIVATE_KEY_FILE = "IDENTITY_PRIVATE_KEY_FILE"
 
+	ENV_PROMETHEUS_SERVICE = "PROMETHEUS_SERVICE"
```

Comment:
we should append _URL to the name to make it clear this setting expects a URL (and not, for example, a hostname or a "host:port" formatted value).

`ENV_PROMETHEUS_SERVICE_URL = "PROMETHEUS_SERVICE_URL"`

Reply:
+1